jekyll/_cci2/manually-starting-machine-runner.adoc

```diff
@@ -0,0 +1,80 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Manually starting machine runner on Linux
```

Comment:
Is this to take the place of the manual machine runner installation on Linux that we currently have here? https://github.com/circleci/circleci-docs/pull/8329/files#diff-23c3aec405f4050173b7a8d02816091669f2a51e840fc82514ab76ab3efef2a7

If so we need to look at the prerequisite steps we have listed there to keep things consistent. We're using reusable snippets quite a lot in these runner docs, and we would like to keep the language consistent through all pages.

I was a bit confused by the different language choice "manually start jobs using machine runner" As opposed to install machine runner?